pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs/multiregion-9node-3region-3azs_test: TestCCLLogic_regional_by_row_cascade failed

pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs/multiregion-9node-3region-3azs_test.TestCCLLogic_regional_by_row_cascade [failed](https://mesolite.cluster.engflow.com/invocations/default/ef9f2eda-b92e-409d-9d96-cebacf7a5081?testReportRun=2&testReportShard=23&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy9tdWx0aXJlZ2lvbi05bm9kZS0zcmVnaW9uLTNhenM6bXVsdGlyZWdpb24tOW5vZGUtM3JlZ2lvbi0zYXpzX3Rlc3Q=) on master @ [d1cef1a1ff36af682125983fcdbc0d15060cb3f1](https://github.com/cockroachdb/cockroach/commits/d1cef1a1ff36af682125983fcdbc0d15060cb3f1):


```
=== RUN   TestCCLLogic_regional_by_row_cascade
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestCCLLogic_regional_by_row_cascade2602215036
    test_log_scope.go:82: use -show-logs to present logs inline
[06:02:22] setting distsql_workmem='66478B';
    logic.go:3152: 
         
        /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs/multiregion-9node-3region-3azs_test_/multiregion-9node-3region-3azs_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/regional_by_row_cascade:67: SELECT count(*) FROM [EXPLAIN (VERBOSE) DELETE FROM great_grandparent WHERE i = 1] WHERE info LIKE '%parallel%';
        expected:
            1
            
        but found (query options: "") :
            0
            
[06:02:23] --- done: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs/multiregion-9node-3region-3azs_test_/multiregion-9node-3region-3azs_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/regional_by_row_cascade with config multiregion-9node-3region-3azs: 13 tests, 1 failures
    logic.go:4491: 
        /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs/multiregion-9node-3region-3azs_test_/multiregion-9node-3region-3azs_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/regional_by_row_cascade:72: error while processing
    logic.go:4491: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs/multiregion-9node-3region-3azs_test_/multiregion-9node-3region-3azs_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/regional_by_row_cascade:72: too many errors encountered, skipping the rest of the input
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestCCLLogic_regional_by_row_cascade2602215036
--- FAIL: TestCCLLogic_regional_by_row_cascade (3.73s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=2</code>
 - <code>shard=23</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_regional_by_row_cascade.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-55607

--- a/pkg/ccl/logictestccl/testdata/logic_test/regional_by_row_cascade
+++ b/pkg/ccl/logictestccl/testdata/logic_test/regional_by_row_cascade
@@ -64,7 +64,7 @@ SELECT crdb_internal.clear_table_stats_cache();
 
 # Only the scan in the main query is parallelized when we don't have stats on
 # the descendant tables.
-query I
+query I retry
 SELECT count(*) FROM [EXPLAIN (VERBOSE) DELETE FROM great_grandparent WHERE i = 1] WHERE info LIKE '%parallel%';
 ----
 1

